server/src/main/java/org/elasticsearch/search/aggregations/bucket/composite/CompositeValuesSourceBuilder.java

```diff
@@ -276,7 +302,7 @@ public String format() {
 
     public final CompositeValuesSourceConfig build(SearchContext context) throws IOException {
         ValuesSourceConfig<?> config = ValuesSourceConfig.resolve(context.getQueryShardContext(),
-            valueType, field, script, null,null, format);
+            valueType, field, script, null, timeZone, format);
```

Comment:
Do you think it'd be cleaner to make a
```
protected ZoneId timeZone()
```
method on this class and override it in the DateHistogramValuesSourceBuilder?

Reply:
I think it would be OK to do that. I will look in detail tomorrow. 
Thank you very much for your review.